### PR TITLE
Add Balance Component to Storybook

### PR DESCRIPTION
**Pull request by colefriedlaender**
Balance Component

### CODE REVIEW

**Review by fre-ben — APPROVED** [review]
LGTM 💯

File: components/Balance/Balance.module.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.container {
+  background-color: var(--text-button-color);
+  height: 80px;
```
Comment (phlgr) [inline components/Balance/Balance.module.css]: Why does this component need a fixed height

File: components/Balance/Balance.module.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  grid-template-rows: 25px 1fr;
+  grid-template-columns: 1fr 1fr;
+  padding-left: 25px;
+  padding-right: 25px;
+  box-shadow: 0 2px 5px;
+}
+.head {
+  color: var(--secundary-color);
+  align-self: center;
+  font-size: 15px;
```
Comment (phlgr) [inline components/Balance/Balance.module.css]: This should be responsive

File: components/Balance/Balance.module.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  grid-column: 1;
+  grid-row: 1;
+}
+.total {
+  font-size: 25px;
```
Comment (phlgr) [inline components/Balance/Balance.module.css]: Same as the above

File: components/Balance/Balance.module.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  font-weight: bold;
+  grid-column: 1;
+  grid-row: 2;
+}
+.returnValue {
+  justify-self: end;
+  font-size: 25px;
```
Comment (phlgr) [inline components/Balance/Balance.module.css]: Same as the above

File: components/Greeting/Greeting.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import styles from "../Greeting/Greeting.module.css";
 export type NameProps = { name: string };
 
-function Button({ name }: NameProps) {
+function Greeting({ name }: NameProps) {
```
Comment (phlgr) [inline components/Greeting/Greeting.tsx:4]: Why is this in here?

**Review by phlgr — CHANGES_REQUESTED** [review]

**Pull request merged**